pkg/controllermanager/controller/shoot/reference/add_test.go

```diff
@@ -81,7 +82,7 @@ var _ = Describe("Add", func() {
 					Expect(p.Update(event.UpdateEvent{ObjectNew: shoot, ObjectOld: oldShoot})).To(BeTrue())
 				})
 
-				It("should return because the audit policy fields changed", func() {
+				It("should return true because the audit policy field changed", func() {
```

Comment:
```
				It("should return true because the DNS fields changed", func() {
```
Please correct the above description as well.

Reply:
No, it's correct only. This test is for `AuditConfig`.

Reply:
I meant https://github.com/gardener/gardener/blob/5abed1bfedd6f2188e5dcec9af161d1b5c286236/pkg/controllermanager/controller/shoot/reference/add_test.go#L74